Jenkinsfile

```diff
@@ -482,11 +482,11 @@ noToolsScalaVersions.each { scalaVersion ->
   quickMatrix.add([task: "test-suite-ecma-script6", scala: scalaVersion, java: mainJavaVersion, testSuite: "testSuite"])
 }
 allJavaVersions.each { javaVersion ->
-  quickMatrix.add([task: "tools-cli-stubs-sbtplugin", scala: "2.10.7", java: javaVersion])
+  quickMatrix.add([task: "tools-cli-stubs-sbtplugin", scala: "2.10.7", sbt_version_override: "0.13.17", java: javaVersion])
```

Comment:
To go with the change below, I'd rather not set `sbt_version_override` here, but change 2.10.7 into 2.12.8. And then change 2.12.8 into 2.10.7 on the line below (the one with `tools-cli-stubs`).

Reply:
Fixed